### ISSUE TITLE
Proposal: Mixin Assignments

### ISSUE THREAD

**Issue by grebaldi** [reduct/assembler#8]
I thought about this for quite a while now and I think it does make sense. Think of the following scenario:

I have a form, like:

``` html
<form data-app="MyFormComponent"></form>
```

MyFormComponent might combine quite a lot of functionality, like some visual effects, validation and printing of validation messages.

Since at least two of these functionalities do not depend on each other, you would have better semantics and decoupling by writing something like this:

``` html
<form data-app="FancyEyeCandyComponent ValidatorComponent"></form>
```

If you really have disjunct functionality (like eye-candy <-> validation) it would just be great to have the option to apply all at once, whithout the need to hard-wire them into each other on JS side.

One could go pretty mad about this one and write something like this:

``` html
<form data-app="FancyEyeCandyComponent ValidatorComponent SendStuffToServerComponent">
</form>
```

Which would be easily interchangeable to:

``` html
<form data-app="FancyEyeCandyComponent ValidatorComponent JustLogStuffComponent">
</form>
```

If these apply in order, you would get interchangeable and functionality-disjunct components, with the positive effect of describing behavior on the fly.

What do you think?

**Comment by Inkdpixels** [comment]
Yeah I also had this in mind a while ago, and it does indeed make a lot of sense. So if you have a some spare time, go for a PR - Or I will implement this as soon I am Done with the ES6 rewrite as seen in #6
